### DISCUSSION TITLE
Longer code on GitHub loads slow and freezes Chrome Mobile, is it something wrong on my end?

### DISCUSSION THREAD

**Discussion by ericoporto** [community/community#23695] [New to GitHub]
<p>My main phone is a Samsung Note9, it’s not that old and usually runs things well. Unfortunately browsing source code is not viable on Chrome Mobile in this phone, since it often freezes the browser.</p>
<p>This doesn’t happen on Firefox Mobile (I use Firefox Nightly). I can reproduce this in 5 different Android Phones with Chrome Mobile - all from 2018-2020.</p>
<p>Is there any chance to fix this on GitHub side?</p>

**Comment by R4356th** [comment]
<p>I am able to view and edit big files on GitHub on Samsung Galaxy J4 and J7 using Chrome without any problem.</p>

**Comment by ericoporto** [comment]
<p>Can you try a code that has more than 8 contributors? I am not sure if it needs this on top of code size to trigger but I still have it.</p>
<p>It has certainly got better over time (this used to be more severe) but it still happens, and it loads blazingly fast on Firefox Nightly. I just find weird that it makes my Chrome sluggish until it loads - I can’t even change screen or hit home until this load happens. I am reporting because I noticed the small improvements lately so it seems to be on the right track.</p>
<p>Conversely, my company firewall on desktop makes the commit messages never load - they are a gray rectangle when I am in that network.</p>

**Comment by riking** [comment, marked as answer]
<p>This is a common issue with Chrome and complex, long pages. Pretty much the only way GitHub could avoid it is to give a raw view with no syntax highlighting or line numbers.</p>
<p>In other words, it’s a Chrome performance bug.</p>

**Comment by ericoporto** [comment]
<p>Thanks for confirming it.</p>
<p>I will keep using Firefox for GitHub on mobile for now.</p>

**Comment by JasonHoku** [comment]
I am suddenly experiencing this on an unchanged file that was working two days ago but now freezes both Chrome and Firefox in a similar manner of partially loading some of the text on the page and then freezing when I click in the code area or try to do a search with ctrl + f

Edit:  https://downdetector.com/status/github/ shows this is potentially a non local incident. If you really need to view code, clicking the "Raw" button before the page freezes is an option.
